<launch>
  <node pkg="complete_mission" type="ego_planner" name="" output="screen">
    <rosparam command="load" file="$(find complete_mission)/config/complete_mission.yaml" />
  </node>
</launch>